commands/build.go

```diff
@@ -90,16 +90,16 @@ func suggestSettingBuilder(logger *logging.Logger) {
 
 func suggestBuilders(logger *logging.Logger) {
 	logger.Info("Suggested builders:\n")
+
 	tw := tabwriter.NewWriter(logger.RawWriter(), 10, 10, 5, ' ', tabwriter.TabIndent)
-	for len(suggestedBuilders) > 0 {
-		n := rand.Intn(len(suggestedBuilders))
-		builders := suggestedBuilders[n]
+	for _, i := range rand.Perm(len(suggestedBuilders)) {
```

Comment:
This replaces the need to mutate `suggestedBuilders` and prevent consecutive tests to fail due to change of state.